swiftsimio/objects.py

```diff
@@ -7,13 +7,12 @@
 
 import unyt
 from unyt import unyt_array
+from unyt.array import multiple_output_operators
 
-# Eventually import of POWER_SIGN_MAPPING will fail because it has been
-# replaced with POWER_MAPPING in a bugfix (unyt PR#231). Then see segment
-# of unyt.array.__array_ufunc__ where it is used and update
-# cosmo_array.__array_ufunc__ accordingly. There is an xfailing test in
-# test_cosmo_array_attrs to catch when this occurs with a test failure.
-from unyt.array import multiple_output_operators, POWER_SIGN_MAPPING
+try:
+    from unyt.array import POWER_MAPPING
+except ImportError:
+    raise ImportError("unyt >=2.9.0 required")
```

Comment:
Change this in the setup.py too.

Reply:
Done, removed testing on python3.6 and updated mentions of the python version in the docs, too.